src/components/Calculator.js

```diff
@@ -0,0 +1,69 @@
+import './Calculator.css';
+
+function Calculator() {
+  return (
+    <div className="container">
+      <div className="btns">
+        <button className="btn" type="button" value="AC">
+          AC
+        </button>
+        <button className="btn" type="button" value="+/-">
+          +/-
```

Comment:
- [ ] In this activity we want you to use one child (custom) component inside the _Calculator_ component. Please improve upon this aspect.

Reply:
Can you please elaborate?

Reply:
Hi @kessie2862 

I think you can create a Button component which hold all the buttons of your app. Then, import and use the Button component you've created inside the Calculator component.  Doing so will keep the component organized and decoupled.

Reply:
Alright. Thanks

Reply:
Hi @kessie2862 

Kindly check carefully in your ReadMe:
![image](https://github.com/kessie2862/Math-Magicians/assets/12745474/fedc743c-5b9f-45dd-b4f2-27e29bbf6ae2)

Reply:
That might be yesterday. I've already fixed all that. 

![readme](https://github.com/kessie2862/Math-Magicians/assets/97234029/82634914-a26a-435e-925a-4df304d143f2)